src/css/cs/items/_kitchensink.scss

```diff
@@ -376,6 +376,19 @@
     }
   }
 
+  // Style the text that appears in the HTMLArea on UC_STATUS_MSG_SBF
+  .uc-status-message {
+    border: 0;
+    box-shadow: none;
+    font-size: 16px;
+    font-weight: bold;
+    margin: 0;
+    outline: none;
```

Comment:
@christianvuerings I'm not sure what you're referring to "this"?  The whole class?

"this allows people to see where the focus is" - do you mean the blue highlight around the textarea?  That is disabled by `outline: none`